app/src/scripts/google-analytics.js

```diff
@@ -64,15 +65,25 @@ module.exports.sendEvent = function event(customObject, ...args) {
   }
 }
 
-module.exports.track = function track(...args) {
+function track(...args) {
   if (window.ga) {
     window.ga(`send`, ...args)
   }
 }
 
-module.exports.setGoogleAnalyticsPage = function track(...args) {
+function setGoogleAnalyticsPage(...args) {
   if (window.ga) {
     window.ga(`set`, `page`, ...args)
     window.ga(`send`, `pageview`)
   }
 }
+
+module.exports = {
+  anonymize,
+  deanonymize,
+  enableGoogleAnalytics,
+  disableGoogleAnalytics,
+  sendEvent,
+  track,
+  setGoogleAnalyticsPage
```

Comment:
Looks good. How can I test this in Bitrise? I forgot the whole flow there. Maybe I still have bookmarked on my Windows. Not sure

Reply:
https://live.browserstack.com/ there is also a chrome extension that will open the current URL right in browserstack

Reply:
cool, I will check this later